src/components/List/Cells/ApplicationNameCell.tsx

```diff
@@ -1,17 +1,16 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
-import { Header } from 'semantic-ui-react'
 import { CellProps } from '../../../utils/types'
+import USER_ROLES from '../../../utils/data/userRoles'
 
-const ApplicationNameCell: React.FC<CellProps> = ({ application }) => (
-  <Link
-    // size="small"
-    // as={Link}
-    to={`/application/${application.serial || 0}`}
-    // content=
-  >
-    {application.name as string}
-  </Link>
-)
+const ApplicationNameCell: React.FC<CellProps> = ({ application, query }) => {
+  const { userRole } = query
+  const linkUrl =
+    userRole === USER_ROLES.REVIEWER
+      ? `/application/${application.serial || 0}/review`
+      : `/application/${application.serial || 0}`
+
```

Comment:
I agree with Andrei, this would be better to be passed on and received here only as a `linkUrl` to be called

Reply:
Okay, I think I'm getting two slightly different suggestions. Andrei suggests having a completely different Cell component, Nicole is suggesting same component, but with "linkUrl" passed in as prop.

I think passing in the link URL sounds better, since it seems overkill to create an additional cell component type for what is essentially a trivial difference between the two cases.

So are we cool if I:
- calculate "linkURL" string in ApplicationsList (while iterating over applications)
- pass it as additional prop (instead of `query`) to cells
- ApplicantNameCell just renders the link it's passed